test/libsolidity/semanticTests/array/copying/array_copy_target_leftover.sol

```diff
@@ -21,4 +21,4 @@ contract c {
 // test() -> 0xffffffff, 0x0000000000000000000000000a00090008000700060005000400030002000100, 0x0000000000000000000000000000000000000000000000000000000000000000
 // gas irOptimized: 223456
 // gas legacy: 328106
-// gas legacyOptimized: 308072
+// gas legacyOptimized: 308192
```

Comment:
This is a bit weird... this is the only case in which this PR regresses on gas costs relative to the latest version of https://github.com/ethereum/solidity/pull/10942 (i.e. a version that does not reuse argument slots after return variables are allocated), but the diff actually looks like it should *improve* as well, not regress :-)...